db/migrate/20211116114852_update_local_authority.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class UpdateLocalAuthority < ActiveRecord::Migration[6.1]
+  class LocalAuthority < ApplicationRecord; end
+
+  def change
```

Comment:
Should this be `up_only` ?

Reply:
probs unnecessary for this but you could also provide a down block to update the columns back to it's original state

Reply:
I'm not sure, wouldn't up be only used for a "new" column? the documentation isn't very clear: https://apidock.com/rails/v6.0.0/ActiveRecord/Migration/up_only

Reply:
ah re the down block - its irreversible as its a content change from what we have in the seeds. I should probably change the seeds too though 🤔

Reply:
Yeah `up_only` just refers to the up migration. Can be any modification but is just run when you do db:migrate and not db:rollback

Reply:
I guess what I meant with the down block is add some code to update the enquiries_paragraph to what it was before the migration would have taken place. But it's unlikely we'll have to revert back to the old copy anyway